geoips/commandline/geoips_list.py

```diff
@@ -662,10 +662,15 @@ class GeoipsList(GeoipsCommand):
     """Top-Level List Command for listing off GeoIPS Artifacts."""
 
     name = "list"
-    generated_classes = CommandClassFactory(
-        GeoipsListSingleInterface,
-        interfaces.__all__,
-    ).classes
+    generated_classes = []
+    for int_name in interfaces.__all__:
```

Comment:
Is this the loop you were going to move outside the class in another PR?

Reply:
This loop is the 'moved loop'. Originally this loop was implemented in ``CommandClassFactory``, however we wanted that class to only generate one class per instantiation. This is where we'll want to keep this loop for the time being, unless we think of a different way to implement this.

Reply:
See #708 for the future of this.